lib/solr_record.rb

```diff
@@ -18,8 +19,8 @@ def cluster
     end
   end
 
-  def ht_items
-    @ht_items ||= cluster.ht_items =
+  def parse_ht_items
```

Comment:
We want to ensure this always happens rather than relying on lazy-loading. If we make a `SolrRecord` we are always going to use its ht items.